### ISSUE TITLE
Another MSVC++ 2010 issue with variable declaration in packager.c

### ISSUE THREAD

**Issue by utelle** [jmcnamara/libxlsxwriter#60]
Since MSVC++ 2010 does not support the C99 standard, variable declaration have to be placed at the beginning of scope block. Therefore `packager.c` doesn't compile properly due to lines 58 and 59 declaring a variable after some executable code:
```
58:    /* Use the native Win32 file handling functions with minizip. */
59:    zlib_filefunc64_def filefunc;
```
This is allowed for C99, but not for C89. Moving these 2 lines  a bit up, for example just **below** line 47
```
47: wchar_t wide_filename[_MAX_PATH + 1] = L"";
```
solves the problem.

**Comment by jmcnamara** [comment]
Fixed on master.

**Comment by jmcnamara** [comment]
Fixed in release 0.4.0. Thanks for the report.
